lambda/mod.ts

```diff
@@ -224,21 +229,67 @@ export interface Lambda {
  *   lo.dispose();
  * }
  * ```
+ *
+ * You can pass JSON serializable values, {@linkcode Expression} or
+ * {@linkcode [eval].RawString|RawString} for the {@linkcode Lambda#notify} or
+ * {@linkcode Lambda#request} arguments.
```

Comment:
Are you sure that `{@linkcode Lambda#notify}` links as expected? Or should I test it via pre-release of this branch?

Reply:
Please test via pre-release.

Reply:
@Milly It seems it doesn't work as expected. We should use `{@link [lambda].notify|notify}` instead.

https://jsr.io/@denops/std@7.1.0-pre.0/doc/lambda/~/add
![CleanShot 2024-08-25 at 22 28 06](https://github.com/user-attachments/assets/fe7e9dd6-da31-4f4e-a16c-e128332cb0fd)